clang/lib/CodeGen/CGExprScalar.cpp

```diff
@@ -1094,6 +1114,27 @@ void ScalarExprEmitter::EmitIntegerTruncationCheck(Value *Src, QualType SrcType,
                 {Src, Dst});
 }
 
+static llvm::Value *EmitIsNegativeTestHelper(Value *V, QualType VType,
```

Comment:
Please extract lambda to function into a separate patch

Reply:
fixed

Reply:
I still see it here, does it needs rebase, or that patch needs to be landed?